doc/src/fpga_api/quick_start/readme.md

```diff
@@ -393,6 +393,79 @@ Done
 
 ```
 
+## Setup IOFS Release1 Bitstream on FPGA PCIe card  ##
+
+Program IOFS Release1 bitstream on FPGA D5005 or N6010 cards and reboot system.
+
+Run command: lspci | grep acc
+
+```3b:00.0 Processing accelerators: Intel Corporation Device af00 (rev 01)```
+
+Number of virtual functions supported by bitstream
+
+``` 
+   cat /sys/bus/pci/devices/0000\:3b\:00.0/sriov_numvfs 
```

Comment:
We can remove the backslash ( \ ) characters before the colons: should be 0000:3b:00.0

Reply:
fixed , replaced \ with :